Apps/frontend/src/components/OnOffButton.svelte

```diff
@@ -24,3 +32,8 @@
   )}
   on:click={handleClick}
 />
+<Tooltip target={button} placement="bottom">
+  {TOOLTIP_BUTTON_OFFOFF_BASE}{$osciEnabled
+    ? TOOLTIP_BUTTON_OFFOFF_OFF
+    : TOOLTIP_BUTTON_OFFOFF_ON}
```

Comment:
Do we really want three labels for this? 
Wouldn't one be good enough? And even if we want to describe the state, two labels would be fine instead of putting the two states together from three labels.

Reply:
Changed it to two.